config/publisher.yml

```diff
@@ -7,3 +7,9 @@ host_to_automatic_api_host:
   - ".malibu"
 domain_mapping:
   subdomain.lvh.me: subdomain
+
+publisher:
```

Comment:
rename to something more generic , like analytics or something.

Reply:
publisher contains all other related config keys